app/service/parsing_svc.py

```diff
@@ -17,11 +17,13 @@ def __init__(self, data_svc):
             parser = await self.data_svc.dao.get('core_parser', dict(ability=x['ability']))
             if parser:
                 if parser[0]['name'] == 'json':
-                    matched_facts = self._json(parser[0], b64decode(x['output']).decode('utf-8'))
+                    matched_facts = parsers._json(parser[0], b64decode(x['output']).decode('utf-8'))
```

Comment:
since we're moving parsers out of this class, let's "promote" them to first-class citizens and change them from private to public functions (i.e., remove the underscore from their names). FANTASTIC idea and execution here though.